src/migrations/20251740066033_user_info.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  await knex.schema.createTable('user_info', table => {
+    table.increments();
```

Comment:
Sorry last one! We don't need an auto_incrementing field here, but we _do_ need:

```suggestion
    table.integer('principal_id').notNullable().primary().references('principals.id')
```

Users are stored in the `principals` table. A principal is either a user, an app or a group.

So we need to re-use the primary key, which is also a foreign key on this table.

Reply:
that makes sense to me for the purposes of table having an id for the user... I also had something similar in my initial PR